tests/Integration/Endpoints/RecommendationsProxyEndpointTest.php

```diff
@@ -0,0 +1,126 @@
+<?php
+/**
+ * Parsely Recommendations API Proxy Endpoint tests.
+ *
+ * @package Parsely\Tests
+ */
+
+declare(strict_types=1);
+
+namespace Parsely\Tests\Integration;
+
+use Parsely\Parsely;
+use Parsely\Endpoints\Recommendations_API_Proxy;
+use WP_Error;
+use WP_REST_Request;
+use WP_REST_Server;
+
+/**
+ * Parsely REST API tests.
+ */
+final class RecommendationsProxyEndpointTest extends TestCase {
+	/**
+	 * Set up globals & initialize the Endpoint.
+	 */
+	public function setUp(): void {
+		parent::setUp();
+		add_filter( 'wp_parsely_enable_recommendations_endpoint', '__return_true' );
+		update_option( 'parsely', array( 'apikey' => 'example.com' ) );
```

Comment:
This will wipe out all of the other Parsely settings, which isn't reflective of real-world happenings.

Please use [`setOptions()` method](https://github.com/Parsely/wp-parsely/blob/ebd84b8d3c124b178b7ca8a8f3136eb9d7b24e73/tests/Integration/TestCase.php#L61) so that the APIkey value is merged in with the rest.